contracts/Comptroller/Diamond/facets/SetterFacet.sol

```diff
@@ -213,6 +219,21 @@ contract SetterFacet is ISetterFacet, FacetBase {
         return __setLiquidationIncentive(newLiquidationIncentiveMantissa);
```

Comment:
`setLiquidationIncentive` should be updated, to include the vToken address. 

The attribute `liquidationIncentiveMantissa` in the `ComptrollerV1Storage` could be renamed to `__oldLiquidationIncentiveMantissaSlot`, including a comment, as we are doing with other deprecated fields.

The attribute `closeFactorMantissa` in the `ComptrollerV1Storage` could be renamed to `__oldCloseFactorMantissaSlot`, including a comment, as we are doing with other deprecated fields.

FacetBase.closeFactorMinMantissa and FacetBase.closeFactorMaxMantissa could be removed

SetterFacet.__setCloseFactor, setCloseFactor and _setCloseFactor could be removed. And the associated event: NewCloseFactor

The associated interfaces should be updated after these changes

Reply:
[6868164](https://github.com/VenusProtocol/venus-protocol/pull/604/commits/6868164b5ef0aca3e25d06eca5b3fb39f4c04abf)
[de84582](https://github.com/VenusProtocol/venus-protocol/pull/604/commits/de84582b68ccc3e36e7ac8b612416f5107c58b14)

Reply:
FacetBase.closeFactorMinMantissa and FacetBase.closeFactorMaxMantissa could be removed

Reply:
[02dc4e7](https://github.com/VenusProtocol/venus-protocol/pull/604/commits/02dc4e734f557052f0d1ca3703812063ff8989b3)